Legacy/bonej/src/main/java/org/bonej/plugins/ConnectedComponents.java

```diff
@@ -979,8 +969,7 @@ private static void expandMap(final List<IntHashSet> map, final int ID,
 	 * @return number of particles in the image
 	 */
 	public int getNParticles() {
-		final int n = nParticles;
-		return n;
+		return nParticles;
 	}
```

Comment:
Is it safe to return a field like this?

Reply:
Yes. It's a primitive field, so value is returned.